package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "podio-js",
-  "version": "1.4.0",
+  "version": "1.4.2",
```

Comment:
Don't increment versions on branches. Only ever do this on master. When you do, use `npm version <major|minor|patch>` which will also create the corresponding tag to keep Github releases in sync with npm releases